main.py

```diff
@@ -166,6 +176,14 @@ def _enrich_project_info_with_costs(projects):
     return projects
 
 
+def _enrich_project_info_with_orgs(client, projects):
+    for project in projects:
+        project['org']=_get_organization(client, project)
```

Comment:
nit: add spaces before and after `=`

Reply:
- space added before and after `=` 14a6234